documentation/modules/auxiliary/scanner/http/wp_fastest_cache_sqli.md

```diff
@@ -0,0 +1,134 @@
+## Vulnerable Application
+
+The vulnerability affects the **WP Fastest Cache** plugin, version **1.2.2** and below, allowing SQL injection via a multipart form.
+
+### Pre-requisites:
+   - **Docker** and **Docker Compose** installed on your system.
+
+### Setup Instructions:
+
+1. **Download the Docker Compose file**:
+   - Here is the content of the **docker-compose.yml** file to set up
+   WordPress with the vulnerable WP Fastest Cache plugin and a MySQL database.
+
+```yaml
+version: '3.1'
+
+services:
+  wordpress:
+    image: wordpress:latest
+    restart: always
+    ports:
+      - 5555:80
+    environment:
+      WORDPRESS_DB_HOST: db
+      WORDPRESS_DB_USER: chocapikk
+      WORDPRESS_DB_PASSWORD: dummy_password
+      WORDPRESS_DB_NAME: exploit_market
+    mem_limit: 512m
+    volumes:
+      - wordpress:/var/www/html
+
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_DATABASE: exploit_market
+      MYSQL_USER: chocapikk
+      MYSQL_PASSWORD: dummy_password
+      MYSQL_RANDOM_ROOT_PASSWORD: '1'
+    volumes:
+      - db:/var/lib/mysql
+
+volumes:
+  wordpress:
+  db:
+```
+
+2. **Start the Docker environment**:
+   - In the directory where you saved the `docker-compose.yml` file, run the following command to start the services:
+
+```bash
+docker-compose up -d
+```
+
+3. **Install WP Fastest Cache Plugin**:
+   - Download the vulnerable version of WP Fastest Cache:
+
+```bash
+wget https://downloads.wordpress.org/plugin/wp-fastest-cache.1.2.1.zip
+```
+
+   - Install the plugin in your running WordPress instance:
+     - Extract the plugin files and copy them to your WordPress container:
+
+```bash
+unzip wp-fastest-cache.1.2.1.zip
+docker cp wp-fastest-cache wordpress:/var/www/html/wp-content/plugins/
+```
+
+   - Navigate to `http://localhost:5555/wp-admin` in your browser and activate the **WP Fastest Cache** plugin in the WordPress admin panel.
+
+4. **Enable Permalinks and Caching**:
```

Comment:
Hello @dledda-r7 did you do this step? Maybe that's why the exploitation is stuck

Reply:
Yes, in the Plugin settings i have marked the "Cache system enable"
and then i've set the permalinks to "post name" inside Wordpress Settings -> Permalinks

Reply:
Okay yeah. But I suspect it might be the sqlidelay then because all the other auxiliary modules had managed to work.

Reply:
do you have a value you suggest?

Reply:
> do you have a value you suggest?
> 
> I tried with 5 seconds and seems the same.

Ok I'll debug. With 5 seconds it's actually not normal yes

Reply:
Hello @dledda-r7, I think I’ve fixed the bug. The verification check works when I send requests to `/`, but the exploitation doesn’t. Additionally, after multiple attempts to rerun the check, it stops working, and I have to disable and re-enable caching for it to work again. The solution is to use `wp-admin.php`, which I removed in a previous commit thinking it wasn't necessary. https://github.com/rapid7/metasploit-framework/blob/0c8135631252c1d23daf015de3e21b7e54e376d6/modules/auxiliary/scanner/http/wp_fastest_cache_sqli.rb#L61